services/blockchain-connector/shared/sdk/blocksUtils.js

```diff
@@ -80,28 +76,46 @@ const loadConfig = async () => {
 	}
 };
 
-const downloadGenesisBlock = async () => {
+// eslint-disable-next-line consistent-return
+const downloadAndValidateGenesisBlock = async (retries = 2) => {
```

Comment:
Please consider wrapping up the entire logic inside of a `try..catch` block.

Reply:
Applied